test/commands/skuid/page/pull.test.ts

```diff
@@ -64,67 +80,79 @@ describe('skuid:page:pull', () => {
         }));
       }
       return Promise.reject(new Error("Unexpected request"));
-    })
-    .stdout()
-    .command(['skuid:page:pull', '--targetusername', 'test@org.com'])
-    .it('runs skuid:page:pull with no page or module specified', ctx => {
-      expect(ctx.stdout).to.contain('Wrote 2 pages to skuidpages');
-    });
+    };
 
-  test
-    .withOrg({ username: 'test@org.com' }, true)
-    .withConnectionRequest(request => {
+    const cmd = new Pull(
+      ["--target-org", testData.username],
+      config
+    );
+
+    const result = await cmd.run();
+    expect(result).to.equal('Wrote 2 pages to skuidpages');
```

Comment:
This is what I'm talking about, the `result` used to be just the empty object, now it's the string of what was logged but it shouldn't be that. We need to somehow access the logger and see what was logged. I did have some success with mocking the function as well and testing what was passed. Just didn't reimplement that.